packages/medusa/src/services/index.ts

```diff
@@ -35,7 +36,7 @@ export { default as ProductService } from "./product"
 export { default as ProductCollectionService } from "./product-collection"
 export { default as ProductTypeService } from "./product-type"
 export { default as ProductVariantService } from "./product-variant"
-import { default as PublishableApiKey } from "./publishable-api-key"
```

Comment:
question: is this intentional?

Reply:
Woups, i missed it. No it is not intentional i ll fix that